tests/keys.test.ts

```diff
@@ -72,8 +85,10 @@ describe.each([{ permission: 'Master' }])('Test on keys', ({ permission }) => {
 
   test(`${permission} key: create key with no expiresAt`, async () => {
     const client = await getClient(permission)
+    const uid = '3db051e0-423d-4b5c-a63a-f82a7043dce6'
```

Comment:
Where does it come from?

Reply:
Its a random number, the same way we create random index names!